WordPress/Classes/Extensions/UIImageView+Gravatar.swift

```diff
@@ -3,6 +3,10 @@ import Foundation
 /// UIImageView Helper Methods that allow us to download a Gravatar, given the User's Email
 ///
 extension UIImageView {
+    /// Default name string for the blavatar
+    private static let BlavatarDefault = "blavatar-default"
```

Comment:
Constants are, by (our) guideline, lowercased. Mind changing this one? (or perhaps, even better, something like this:

extension UIImageView {
    struct Blavatar {
        static let defaultImageName = "blavatar-default"
        static let defaultSize = 40
    }
}
```